.github/workflows/format.yml

```diff
@@ -18,6 +20,8 @@ jobs:
           done
       - name: Verify formatting
         run: flutter format --set-exit-if-changed packages
+      - name: Analyze the plugin's Dart code
+        run: tools/run_command.py analyze --run-on-changed-packages --base-sha $(git rev-parse HEAD^)
```

Comment:
The format step runs for all packages while the analyze step only runs for changed packages. Why different.

Analyzing packages doesn't take much time (unlike the build job). We may just run `dart analyze` for all packages for simplicity.

Reply:
> The format step runs for all packages while the analyze step only runs for changed packages. Why different.

OK. I basically preferred to analyze only the packages that have changed, but your opinion is also good. I will change this.

>  We may just run dart analyze for all packages for simplicity.

With Dart, more documentation is needed for each dart file's public API. (ref [this](https://stackoverflow.com/questions/65308789/why-is-flutter-analyze-different-from-dart-analyze))

If you don't mind we can change to `dart` from `flutter` later. otherwise I think we have to do more documentation about API before this PR